app/src/main/java/io/github/droidkaigi/confsched2017/service/helper/OverlayViewManager.java

```diff
@@ -0,0 +1,86 @@
+package io.github.droidkaigi.confsched2017.service.helper;
+
+import android.content.Context;
+import android.content.ContextWrapper;
+import android.graphics.PixelFormat;
+import android.support.v4.view.GravityCompat;
+import android.view.ContextThemeWrapper;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.WindowManager;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import io.github.droidkaigi.confsched2017.R;
+import uk.co.chrisjenx.calligraphy.CalligraphyContextWrapper;
+
+/**
+ * @author KeithYokoma
+ */
+@Singleton
```

Comment:
This `@Singleton` annotation does nothing.
Currently `OverlayViewManager` is provided by `ServiceModule`, so we should add `@ServiceScope` to the provider method if you want to control the lifecycle of this class instance.

Or if `Application` instance can be used for `Context` argument of this class constructor, we can simply remove provider of this class.